app/models.py

```
@@ -1695,74 +1695,49 @@ def _seed_default_class_features(mapper, connection, target):
     # Relationships
     teacher = db.relationship('User', foreign_keys=[user_id], backref=db.backref('announcements', lazy='dynamic', passive_deletes=True))
-    system_admin = db.relationship('User', foreign_keys=[system_admin_id], backref=db.backref('sysadmin_announcements', lazy='dynamic', passive_deletes=True))
-    target_teacher = db.relationship('User', foreign_keys=[target_teacher_id], backref=db.backref('targeted_announcements', lazy='dynamic', passive_deletes=True))
-
-    # Indexes
-    __table_args__ = (
-        db.Index('ix_announcements_audience_type', 'audience_type', 'is_active'),
-        db.Index('ix_announcements_system_admin', 'system_admin_id', 'is_active'),
-    )
 
     def __repr__(self):
```

Ruff (0.16.1)

[warning] Missing return type annotation for special method `__repr__`

Add return type annotation: `str`

(ANN204)